packages/app-mobile/components/BottomDrawer.tsx

```diff
@@ -48,6 +48,8 @@ const useStyles = (theme: ThemeStyle) => {
 				gap: 8,
 				flexDirection: 'row',
 				flexWrap: 'wrap',
+				flexShrink: 1,
+				flexGrow: 1,
```

Comment:
This change was needed for wrapping to work on web. Without it, the "new note" popup's size was always the maximum of 400px (even on very small screens).